ocaml/runtime/caml/mlvalues.h

```diff
@@ -251,14 +251,18 @@ CAMLextern value caml_get_public_method (value obj, value tag);
    to the environment part. */
 #ifdef ARCH_SIXTYFOUR
 #define Arity_closinfo(info) ((intnat)(info) >> 56)
-#define Start_env_closinfo(info) (((uintnat)(info) << 8) >> 9)
-#define Make_closinfo(arity,delta) \
-  (((uintnat)(arity) << 56) + ((uintnat)(delta) << 1) + 1)
+#define Start_env_closinfo(info) (((uintnat)(info) << 9) >> 10)
+#define Is_last_closinfo(info) (((uintnat)(info) << 8) >> 63)
+#define Make_closinfo(arity,delta,is_last) \
+  (((uintnat)(arity) << 56) + ((uintnat)(is_last) << 55) \
+    + ((uintnat)(delta) << 1) + 1)
```

Comment:
The comment above needs to be updated to explain the new format.

Reply:
Fixed